[0m22:14:30.277066 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DDB754E650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DDB754E7D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DDB754E510>]}


============================== 22:14:30.285240 | 350bcb09-4e61-41f8-b3af-12e06dbb3370 ==============================
[0m22:14:30.285240 [info ] [MainThread]: Running with dbt=1.9.3
[0m22:14:30.286267 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\mofob\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt init focal_dbt', 'send_anonymous_usage_stats': 'True'}
[0m22:14:30.347030 [debug] [MainThread]: Starter project path: C:\Users\mofob\AppData\Local\Programs\Python\Python311\Lib\site-packages\dbt\include\starter_project
[0m22:14:30.427598 [info ] [MainThread]: 
Your new dbt project "focal_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m22:14:30.428597 [info ] [MainThread]: Setting up your profile.
[0m22:15:18.422090 [info ] [MainThread]: Profile focal_dbt written to C:\Users\mofob\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m22:15:18.425092 [debug] [MainThread]: Command `dbt init` succeeded at 22:15:18.424093 after 48.23 seconds
[0m22:15:18.425092 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DDB0B88650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DDB0AE71D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DDB0AE6650>]}
[0m22:15:18.426131 [debug] [MainThread]: Flushing usage events
[0m22:15:20.913397 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m22:15:31.407356 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA924DFE50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA9230D590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA924EECD0>]}


============================== 22:15:31.410355 | f98b5887-bc14-44ea-b3dc-d4d044c04d31 ==============================
[0m22:15:31.410355 [info ] [MainThread]: Running with dbt=1.9.3
[0m22:15:31.411355 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': 'C:\\Users\\mofob\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:15:31.433049 [info ] [MainThread]: dbt version: 1.9.3
[0m22:15:31.434050 [info ] [MainThread]: python version: 3.11.5
[0m22:15:31.434050 [info ] [MainThread]: python path: C:\Users\mofob\AppData\Local\Programs\Python\Python311\python.exe
[0m22:15:31.435050 [info ] [MainThread]: os info: Windows-10-10.0.26100-SP0
[0m22:15:31.508260 [info ] [MainThread]: Using profiles dir at C:\Users\mofob\.dbt
[0m22:15:31.509270 [info ] [MainThread]: Using profiles.yml file at C:\Users\mofob\.dbt\profiles.yml
[0m22:15:31.510308 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\mofob\OneDrive\Desktop\PersonalProjects\focal_pipeline\dbt_project.yml
[0m22:15:31.510308 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m22:15:31.511307 [debug] [MainThread]: Command `dbt debug` failed at 22:15:31.511307 after 0.18 seconds
[0m22:15:31.512307 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA92540F50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA925E8090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA924DEBD0>]}
[0m22:15:31.512307 [debug] [MainThread]: Flushing usage events
[0m22:15:33.329796 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m22:15:57.568159 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C3322DF90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C32F5AA10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C3322DC50>]}


============================== 22:15:57.571156 | a9f9418c-fe4e-4813-8ca3-b30e69cacc95 ==============================
[0m22:15:57.571156 [info ] [MainThread]: Running with dbt=1.9.3
[0m22:15:57.572156 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\mofob\\.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m22:15:57.597916 [info ] [MainThread]: dbt version: 1.9.3
[0m22:15:57.598918 [info ] [MainThread]: python version: 3.11.5
[0m22:15:57.599919 [info ] [MainThread]: python path: C:\Users\mofob\AppData\Local\Programs\Python\Python311\python.exe
[0m22:15:57.600978 [info ] [MainThread]: os info: Windows-10-10.0.26100-SP0
[0m22:15:57.673989 [info ] [MainThread]: Using profiles dir at C:\Users\mofob\.dbt
[0m22:15:57.674993 [info ] [MainThread]: Using profiles.yml file at C:\Users\mofob\.dbt\profiles.yml
[0m22:15:57.674993 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\mofob\OneDrive\Desktop\PersonalProjects\focal_pipeline\dbt_project.yml
[0m22:15:57.676001 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m22:15:57.677406 [debug] [MainThread]: Command `dbt debug` failed at 22:15:57.677406 after 0.17 seconds
[0m22:15:57.677406 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C3327D4D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C3321EB90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C33283410>]}
[0m22:15:57.678421 [debug] [MainThread]: Flushing usage events
[0m22:15:59.973919 [debug] [MainThread]: An error was encountered while trying to flush usage events
